src/Renci.SshNet/ScpClient.cs

```diff
@@ -21,8 +21,8 @@ public partial class ScpClient : BaseClient
         /// Gets or sets the operation timeout.
         /// </summary>
         /// <value>
-        /// The timeout to wait until an operation completes. The default value is negative
-        /// one (-1) milliseconds, which indicates an infinite time-out period.
+        /// The timeout to wait until an operation completes. The default value is <see cref="SshNet.Session.InfiniteTimeSpan"/>

```

Comment:
Please revert this. I do not plan on exposing **Session.InfiniteTimeSpan** publicly, and as such we should not mention it in our docs.